src/api/schema/components/mod.rs

```diff
@@ -79,6 +87,17 @@ impl filter::CustomFilter<Component> for ComponentsFilter {
     }
 }
 
+impl sort::SortableByField<ComponentsSortFieldName> for Component {
```

Comment:
To make a type sortable, we implement `sort::SortableByField<T>`. This differs from stdlib comparison traits by receiving a `sort::SortField<T>`, to determine the specific field being sorted.